dvc/repo/experiments/executor/base.py

```diff
@@ -657,19 +659,22 @@ def _repro_dvc(  # noqa: C901
             else:
                 os.chdir(dvc.root_dir)
 
+            args_path = os.path.join(dvc.tmp_dir, cls.PACKED_ARGS_FILE)
+            if os.path.exists(args_path):
+                _, kwargs = cls.unpack_repro_args(args_path)
+            # set env vars using saved config; needed for dvclive to read config
+            for k, v in kwargs.get("run_env", {}).items():
+                os.environ[k] = v
+            dvc_studio_config = get_studio_config(dvc.config.get("studio"))
```

Comment:
The comment is misleading, we don't need to set the env vars here for dvclive.

As long as the env vars are present in `run_env`, they will be exported by DVC and accessible to DVCLive in the stage:

https://github.com/iterative/dvc/blob/be0a1a4a12956248a196d59ead5324486ae5ef55/dvc/stage/run.py#L46

Reply:
Updated the comments. PTAL.